src/parser/mage/arcane/modules/features/ArcanePower.js

```diff
@@ -153,8 +170,24 @@ class ArcanePower extends Analyzer {
     return 0;
   }
 
+  get requiredChecks() {
```

Comment:
Could you add some comments here as to how you are getting the numbers? Or, change the conditional logic to make more sense as you read the code. Something like:
```javascript
let requiredChecks = 4;
if(this.hasOverpowered) {
  requiredChecks -= 1; // Don't check for low mana if overpowered is talented
}
if(!this.hasRuneOfPower) {
  requiredChecks -= 2; // Don't check for RoP delayed or missing if not talented
}
// etc...
```